concepticondata/conceptlists/Huisman-2019-110.tsv

```diff
@@ -0,0 +1,111 @@
+Huisman-2019-110-69	69	to walk	to walk	歩く	aruku		1443	WALK														
+Huisman-2019-110-70	70	to come	to come	来る	kuru		1446	COME														
+Huisman-2019-110-71	71	to lie down	to lie down	寝る	neru		215	LIE DOWN														
+Huisman-2019-110-72	72	to sit	to sit	座る	suwaru		1416	SIT														
+Huisman-2019-110-73	73	to stand	to stand	立つ	tatsu		1442	STAND														
+Huisman-2019-110-74	74	to give1	to give	遣る	yaru		1447	GIVE														
+Huisman-2019-110-75	75	to give2	to give	呉れる	kureru	respectful																
+Huisman-2019-110-76	76	to say	to say	言う	iu		1458	SAY														
+Huisman-2019-110-77	77	sun	sun	日	hi		1343	SUN														
+Huisman-2019-110-78	78	moon	moon	月	tsuki		1313	MOON														
+Huisman-2019-110-79	79	star	star	星	hoshi		1430	STAR														
+Huisman-2019-110-80	80	water	water	水	mizu		948	WATER														
+Huisman-2019-110-81	81	rain	rain	雨	ame		658	RAIN (PRECIPITATION)														
+Huisman-2019-110-82	82	stone	stone	石	ishi		857	STONE														
+Huisman-2019-110-83	83	sand	sand	砂	suna		671	SAND														
+Huisman-2019-110-84	84	soil	soil	土	tsuchi		1228	EARTH (SOIL)														
+Huisman-2019-110-85	85	cloud	cloud	雲	kumo		1489	CLOUD														
+Huisman-2019-110-86	86	smoke	smoke	煙	kemuri		778	SMOKE (EXHAUST)														
+Huisman-2019-110-87	87	fire	fire	火	hi		221	FIRE														
+Huisman-2019-110-88	88	ash	ash	灰	hai		646	ASH														
+Huisman-2019-110-89	89	to burn	to burn	燃える	moeru		2102	BURN														
+Huisman-2019-110-90	90	path	path	道	michi		2252	PATH														
+Huisman-2019-110-91	91	mountain	mountain	山	yama		639	MOUNTAIN														
+Huisman-2019-110-92	92	red1	red	赤	aka	noun																
+Huisman-2019-110-93	93	red2	red	赤い	akai	adjective	156	RED														
+Huisman-2019-110-94	94	blue1	blue	青	ao	noun																
+Huisman-2019-110-95	95	blue2	blue	青い	aoi	adjective	837	BLUE														
+Huisman-2019-110-96	96	yellow1	yellow	黄色	kiiro	noun																
+Huisman-2019-110-97	97	yellow2	yellow	黄色い	kiiroi	adjective	1424	YELLOW														
+Huisman-2019-110-98	98	white1	white	白	shiro	noun																
+Huisman-2019-110-99	99	white2	white	白い	shiroi	adjective	1335	WHITE														
+Huisman-2019-110-100	100	black1	black	黒	kuro	noun																
+Huisman-2019-110-101	101	black2	black	黒い	kuroi	adjective	163	BLACK														
+Huisman-2019-110-102	102	night	night	夜	yoru		1233	NIGHT														
+Huisman-2019-110-103	103	hot	hot	暑い	atsui		1286	HOT														
+Huisman-2019-110-104	104	cold	cold	寒い	samui		1287	COLD														
+Huisman-2019-110-105	105	full	full	いっぱい	ippai		1429	FULL														
+Huisman-2019-110-106	106	new	new	新しい	atarashii		1231	NEW														
+Huisman-2019-110-107	107	good	good	良い	yoi		1035	GOOD														
+Huisman-2019-110-108	108	round	round	丸い	marui		1395	ROUND														
+Huisman-2019-110-109	109	to dry	to dry	乾く	kawaku		1398	DRY														
```

Comment:
Not sure here, but I don't think its the property but rather the action of 'becoming dry', e.g. `1786/DRY UP`?